### DISCUSSION TITLE
how to fix download path on existing torrent?

### DISCUSSION THREAD

**Discussion by tryoutuser** [rakshasa/rtorrent#1181] [Q&A]
hello.

i want to ask how to fix download path on existing torrents i have?

i'm migrating from way older rtorrent on ubuntu 14 and ubuntu 16 i believe it was before v0.9.7.
now i have successfully set up the new version on debian 11, with new path required inside `rtorrent` folder including the session and watch.

i have moved all my downloaded files into respective folder but when i open rutorrent frontend i've got a lot of paused or pausing state torrents.

how do i edit the torrents to move the target download path to the new location?

is it in `sessions` directory? or is it in `rutorrent/share/torrents` directory?

**Comment by chlooriding** [comment]
Hi, I don't know rutorrent, but you can either on rtorrent terminal interface (if you have access to it) type ctrl+o and the new path or better using xmlrpc (you have it if you use rutorrent):

xmlrpc localhost d.directory.set infohash "newpath"

Can't remember if a multi-file torrent needs the correct folder name appended to your new path.

Ideally (even more so in case you have lot of torrents) I would rather build a simple script to:
- loop through all the downloads: xmlrpc localhost download_list
- get the current path (the wrong one): xmlrpc localhost d.directory
- substitute the common part, and apply to the infohash (command above).

hope this helps
M

**Reply by tryoutuser** [reply]
@chlooriding thank you for answering. but when i press ctrl+o the path shown is already the correct path :
![image](https://user-images.githubusercontent.com/35561383/190650422-6417a36c-48cc-4262-9178-6f3e33d970cf.png)

so is there no way to replace or edit the target path? by editing the torrent file maybe or something?

**Comment by chlooriding** [comment]
When you press ctrl+o it shows you the default download path you set in .rtorrent.rc config file (whichever the target path of the download). If you press enter and the download files physically are located on that path then it should rehash and start.

**Reply by tryoutuser** [reply]
oh i have to enter or select one torrent and then ctrl+o?

**Reply by chlooriding** [reply]
select a torrent > ctrl+o > (type the correct path, if needed) > press enter.

**Reply by tryoutuser** [reply]
weird
when i select or enter to respective torrent that is paused/pausing state and i press ctrl+o it still shows to currently set download path in rc config file
